tests/test_options.py

```diff
@@ -146,9 +146,9 @@ def test_multiple_default_help(runner):
     @click.command()
     @click.option("--arg1", multiple=True, default=('foo', 'bar'),
                   show_default=True)
-    @click.option("--arg2", multiple=True, default=(1, 2), type=int,
+    @click.option("--arg2", multiple=True, default=(1, 2), type=(int,int),
```

Comment:
I mean we probably shouldn't break backwards compat like that. Changing it back still makes the tests pass it seems.

I'm currently playing around with your PR locally, to see if there's a better way of fixing this bug.